src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -3,207 +3,109 @@
+    return layoutInfo;
 }
 
-function handleActiveTableChange (tbl_id, images) {
-    // check for catalog or meta images
-
-    const showFits= shouldShowFits();
-    var showImages= showFits;
-
-    if (!tbl_id) {
-        images = clone(images, {showMeta: false, showCoverage: false, showFits, metaDataTableId: null});
-        return [showFits, images];
-    }
-
-    const tblGroup= findGroupByTblId(tbl_id);
-    if (!tblGroup) return [showImages, images];
-    const tblList= getTblIdsByGroup(tblGroup);
-    if (isEmpty(tblList)) return [showImages, images];
-
-    const anyHasCatalog= hasCatalogTable(tblList);
-    const anyHasMeta= hasMetaTable(tblList);
-
-
-    if (!anyHasCatalog && !anyHasMeta) {
-        images = clone(images, {showMeta: false, showCoverage: false, showFits, metaDataTableId: null});
-        return [showFits, images];
-    }
-
-
-
-    if (hasCatalogTable(tblList)) {
-        images = clone(images, {showCoverage: true, showFits});
-        showImages = true;
-    }
+function handleTableActive(layoutInfo, action) {
+    return layoutInfo;
+}
 
-    if (hasMetaTable(tblList)) {
-        const metaTableId= isMetaDataTable(tbl_id) ? tbl_id : findFirstMetaTable(tblList);
-        images = clone(images, {showMeta: true, showFits, metaDataTableId:metaTableId});
-        showImages = true;
-    }
-    else {
-        images = clone(images, {showMeta: false, showFits, metaTableId:null});
+function handleTableHighlight(layoutInfo, action) {
+    const {tbl_id} = action.payload;
+    if (tbl_id === PHASE_FOLDED) {
+        const webPlotReq = getWebPlotRequest(tbl_id);
+        const plotId = webPlotReq.plotId;
+        dispatchPlotImage({plotId, wpRequest:webPlotReq, viewerId:IMG_VIEWER_ID});
     }
-    return [showImages, images];
 }
 
-const hasCatalogTable= (tblList) => tblList.some( (id) => isCatalogTable(id) );
-const hasMetaTable= (tblList) => tblList.some( (id) => isMetaDataTable(id) );
-const findFirstMetaTable= (tblList) => tblList.find( (id) => isMetaDataTable(id) );
-const shouldShowFits= () => !isEmpty(getViewerPlotIds(getMultiViewRoot(), DEFAULT_FITS_VIEWER_ID));
-
-
-
-function handleNewImage(action, images) {
-    var ignore = false;
-    const {viewerId, plotGroupId} = action.payload || {};
-    if (viewerId === META_VIEWER_ID) {
-        // select image meta tab when new images are added.
-        images = clone(images, {selectedTab: 'meta', showMeta: true});
-    } else if (viewerId === DEFAULT_FITS_VIEWER_ID) {
-        // select image tab when new images are added.
-        images = clone(images, {selectedTab: 'fits', showFits: true});
-    } else {
-        ignore = true;
+function getWebPlotRequest(tbl_id) {
+    const tableModel = getTblById(tbl_id);
+    const ra = getCellValue(tableModel, tableModel.highlightedRow, 'ra');
+    const dec = getCellValue(tableModel, tableModel.highlightedRow, 'dec');
+    // return makeWebPlotRequest(tbl_id);  should use the commented code below to retrieve wise images.
```

Comment:
Here we should get 'frame_id' for WISE (don't know for PTF??) and build the plot request. 
Actually, would be ideal to distinguish here the mission so we can build the request depending on it. Do you see this information coming from table metadata?